package.json

```diff
@@ -45,7 +45,7 @@
     "del": "^4.1.1",
     "express": "^4.17.1",
     "html-entities": "^1.3.1",
-    "http-proxy-middleware": "0.19.1",
+    "http-proxy-middleware": "1.0.4",
```

Comment:
How you use `require` changed in http-proxy-middleware version 1.0.0. Looks like you didn't update those statements.

https://github.com/chimurai/http-proxy-middleware/releases/tag/v1.0.0

v0.x.x with implicit require

```js
const proxy = require('http-proxy-middleware');
```

v1.0.0 with explicit require

```js
const { createProxyMiddleware } = require('http-proxy-middleware');
```

Reply:
Wondering why this package's version is locked - does need to stay locked?

Reply:
probably, I changed locked version to the latest version and locked it.

Reply:
alright, I removed version locking and fixed `http-proxy-middleware` usage